doc/user/shared-content/schema-changes-in-progress.md

```diff
@@ -0,0 +1,7 @@
+{{< note >}}
+
+Work to more smoothly support ddl changes to upstream tables is currently in
+progress. The work introduces the ability to re-ingest the same upstream table
+under a new schema.
```

Comment:
can also put in "and switch over" to the end of the sentence.

Reply:
and switch over without downtime